content/billing/scout-billing.md

```diff
@@ -6,92 +6,94 @@ keywords: payments, billing, subscription, scout
 
-In this section, learn how to buy Docker Scout Team in Docker Hub for your personal account or for an organization. To buy Docker Scout Business, [contact sales](https://www.docker.com/products/docker-scout/).
-
-## Personal account
-
-1. Select your avatar in the top-right corner of Docker Hub.
-2. From the drop-down menu select **Billing**.
-3. In the **Plan** section, navigate to the **Docker Scout** plan section. The default plan is **Docker Scout Free**. 
-4. Next to the plan name, select **Buy now**.
-5. From the **Purchase Scout Team** drop-down menu, under **Your Account**, select the name of your personal account.
-6. This redirects you to the payment processing page. Enter your email if this field isn't pre-populated. Then, enter your payment information.
-7. Update the quantity of repositories you want to use with Docker Scout. See [Docker Scout subscriptions](../subscription/scout-details.md) to learn more about the number of repositories available for each plan.
-8. Select annual or monthly billing.
-9. Select **Subscribe**.
-10. Subscribing redirects you back to the billing page in Docker Hub, where you can find your active Docker Scout plan.
-
-Once your purchase is complete, you receive a confirmation email and a copy of your invoice. 
-
-## Organization
-
-1. In Docker Hub, select **Organizations**. 
-2. Select the organization namespace. 
-3. Select the **Billing** tab.
-4. In the **Plan** section, navigate to the **Docker Scout** plan section. The default plan is **Docker Scout Free**. 
-5. Next to the plan name, select **Buy now**.
-6. From the **Purchase Scout Team** drop-down menu, select the name of your organization.
-
-    >**Note**
-    >
-    > If you're an organization with multiple teams, a Docker Scout Business plan may be better. [Contact sales](https://www.docker.com/products/docker-scout/) to learn more.
-
-7. This redirects you to the payment processing page. Enter your email if this field isn't pre-populated. Then, enter your payment information.
-8. Update the quantity of repositories you want to use with Docker Scout. See [Docker Scout subscriptions](../subscription/scout-details.md) to learn more about the number of repositories available for each plan.
-9. Select annual or monthly billing.
+In this section, learn how to buy and manage a Docker Scout Team subscription for your personal account or for an organization. If you're an organization with multiple teams, a Docker Scout Business plan may be better. To buy Docker Scout Business, [contact sales](https://www.docker.com/products/docker-scout/#contact-sales).
+
+> **Note**
+>
+> You need to be an owner to manage billing for an organization.
+>
```

Comment:
Do you think we should capture this in the roles and permissions table for Docker Scout? It's already covered in the organization table. So maybe it isn't necessary to duplicate.

https://docs.docker.com/security/for-admins/roles-and-permissions/#docker-scout

Reply:
Good question! I think it makes sense just listing it in the org management table since we don't specifically say Core billing etc. but this was written before the product ecosystem started to grow. @jonnyrobots what do you think?